notify_center.go

```diff
@@ -176,7 +181,7 @@ func (nc *NotifyCenter) startConsumerGroupEvaluator(group string, cluster string
 		nc.groupLock.RUnlock()
 
 		// Send requests for group status - responses are handled by the main loop (for now)
-		storageRequest := &RequestConsumerStatus{Result: nc.resultsChannel, Cluster: cluster, Group: group}
+		storageRequest := &RequestConsumerStatus{Result: nc.resultsChannel, Cluster: cluster, Group: group, Showall: true}
```

Comment:
Actually all notifiers already have `threshold` parameter which determines whether message/notification should be ignored. This `showAll` flag is used on a different level - consumer group evaluation which informs notifiers when it is necessary.

I do agree with you that it should not be enabled when it is not required. Therefore, I added a commit which enables `showAll` flag only when `graphiteNotifier` is enabled.